packages/design-system/src/components/select.tsx

```diff
@@ -49,6 +51,9 @@ const StyledTrigger = styled(SelectPrimitive.SelectTrigger, {
 const StyledIcon = styled(SelectPrimitive.Icon, {
   display: "inline-flex",
   alignItems: "center",
+  justifyContent: "center",
+  height: "100%",
+  padding: "calc($space$1 / 2)",
```

Comment:
If our spacing scale doesn't work, let's add a `todo` and fix it instead of adding more of `calc`.